android/src/main/jni/OnLoad.cpp

```diff
@@ -0,0 +1,12 @@
+#include <fbjni/fbjni.h>
+
+#include "RNGestureHandlerComponentsRegistry.h"
+
+JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *vm, void *) {
+  return facebook::jni::initialize(vm, [] {
+    // TODO: dvacca ramanpreet unify this with the way
```

Comment:
why?

Reply:
Removed in 3f76094bf94748de6851c4cdde1db9674009024d